src/resolvers/company/companyResolver.ts

```diff
@@ -145,12 +148,19 @@ export const createCompanyAdmin = async (parent: never, { firstName, lastName, e
     }
 
     if (!user) {
-      user = await Users.create({ firstName, lastName, email, password, role, address, company });
+      user = await Users.create({ firstName, lastName, role, address, company });
     }
 
     company.admins!.push(user._id);
 
     await company.save();
 
+    const salt = await bcrypt.genSalt(10);
```

Comment:
could this be a utillity that is then used in companyResolver and authResolver instead of duplicate code?

Reply:
Ja, helt klart, ville være smartere

Reply:
Men kan være det skal laves på en ny branch ud fra dev, så der ikke er noget der bliver ødelagt?

Reply:
jo det kan godt køres bagefter.